webapp-django/crashstats/crashstats/models.py

```diff
@@ -107,6 +107,18 @@ class Meta:
         unique_together = ('vendor_hex', 'adapter_hex')
 
 
+class Platform(models.Model):
```

Comment:
The original table name is `os_names`. The API endpoint was Platform, the middleware model is named "Platform".

"Windows 10" is an operating system name, but the table holds "Windows" which is more of a platform than a specific operating system. 

Given that, I went with "Platform".